.github/workflows/main.yml

```diff
@@ -18,7 +18,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout the code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
+      - name: Setup Java JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11

```

Comment:
**question**: Why do we need to add this part?

Reply:
I add this part to be sure WKF don't use latest version of Java.